brave_domains/BUILD.gn

```diff
@@ -6,11 +6,19 @@
 import("//build/buildflag_header.gni")
 import("buildflags.gni")
 
-# TODO(petemill): Default target for this project could instead be some cpp
-# helpers which create domain names for consumers for the correct environment
-# and can vary based on dynamic cli param overrides.
+source_set("brave_domains") {
+  sources = [
+    "service_domains.cc",
```

Comment:
we now have `brave_domains`, `service_domains`, `brave_services_domain` and `services_domain`. Seems a bit confusing.

Reply:
We have `brave_domains::GetServiceDomain()` (which gets the value of `brave_service_X_domain` vars). I don't see mention of your fourth example (`services_domain`).

It makes sense if "brave domains" is a superset of "brave service domains". But, I'm open to suggestions.